docs/index.md

```diff
@@ -56,10 +62,10 @@ data (generated by the feature store) and code from the source control (Git).
 the API handling, data preparation/enrichment, model serving, ensembles, driving and measuring actions, etc.
 - **Real-Time monitoring and retraining** &mdash; monitors data, models, and production components and provides a feedback loop for exploring production data, identifying drift, alerting on anomalies or data quality issues, triggering re-training jobs, measuring business impact, etc.
 
-While each of those steps may be independent, they still require tight integration. For example:
-- The training jobs need to obtain features from the feature store and update the feature store with metadata, which will be used in the serving or monitoring.
-- The real-time pipeline needs to enrich incoming events with features stored in the feature store, and may use feature metadata (policies, statistics, schema, etc.) to impute missing data or validate data quality.
-- The monitoring layer must collect real-time inputs and outputs from the real-time pipeline and compare it with features data/metadata from the feature store or model metadata generated by the training layer, and it needs to write all the fresh production data back to the feature store so it can be used for various tasks such as data analysis, model re-training (on fresh data), model improvements.
+While each of those layers is independent, the integration provides much greater value and simplicity. For example:
+- The training jobs obtain features from the feature store and update the feature store with metadata, which will be used in the serving or monitoring.
+- The real-time pipeline enrich incoming events with features stored in the feature store, and may use feature metadata (policies, statistics, schema, etc.) to impute missing data or validate data quality.
+- The monitoring layer collect real-time inputs and outputs from the real-time pipeline and compare it with features data/metadata from the feature store or model metadata generated by the training layer, and it writes all the fresh production data back to the feature store so it can be used for various tasks such as data analysis, model re-training (on fresh data), model improvements.
```

Comment:
The monitoring layer **collects** real-time inputs and outputs from the real-time pipeline and **compares them** with the **features** data/metadata from the feature **store,** or model metadata generated by the training **layer. It** writes all the fresh production data back to the feature store so it can be used for various tasks such as data analysis, model re-training (on fresh data), model improvements.